core/translate/planner.rs

```diff
@@ -504,8 +508,9 @@ fn parse_from(
 
     let first_table = match *from.select.unwrap() {
         ast::SelectTable::Table(qualified_name, maybe_alias, _) => {
-            let Some(table) = schema.get_table(&qualified_name.name.0) else {
-                crate::bail_parse_error!("Table {} not found", qualified_name.name.0);
+            let normalized_qualified_name = normalize_ident(qualified_name.name.0.as_str());
+            let Some(table) = schema.get_table(&normalized_qualified_name) else {
```

Comment:
this is also needed in `parse_join()` right?

Reply:
Good catch. Handled it there + added a test.